src/components/common/Extensions/NotificationBanner/NotificationBanner.tsx

```diff
@@ -56,39 +55,45 @@ const NotificationBanner: FC<PropsWithChildren<NotificationBannerProps>> = ({
         {children && (
           <div
             className={clsx('text-sm max-w-[50rem] mt-1.5', {
-              'text-red-400': status === 'error',
+              'text-negative-400': status === 'error',
             })}
           >
             {children}
           </div>
         )}
       </div>
-      <div
-        className={clsx(styles.actionWrapper, 'text-4', {
-          'ml-0 md:ml-0 md:self-center': isAlt,
-          'ml-6 md:ml-2': !isAlt,
-        })}
-      >
-        {(() => {
-          switch (action?.type) {
-            case 'copy': {
-              return <CopyUrl actionText={action.copyContent} />;
-            }
-            case 'redirect': {
-              return <Link to={action.href}>{actionText}</Link>;
+      {action && (
+        <div
+          className={clsx(
+            `[&_a]:underline [&_button]:underline [&_a]:hover:no-underline
+        [&_button]:hover:no-underline mt-2 md:mt-0 text-4`,
```

Comment:
Please add hover on desktop only. You can also add those styles directly to buttons and anchors. Please add className prop if it's missing